js/script.js

```diff
@@ -0,0 +1,165 @@
+const newPlayer = (name, symbol, turn) => {
+  const playerName = name;
+  const token = symbol;
+  const playerTurn = turn;
+  return { playerName, token, playerTurn };
+};
+
+const gameBoard = () => {
+  let board;
+
+  const init = () => {
+    board = [null, null, null, null, null, null, null, null, null];
+    document.getElementById('cell0').innerHTML = '';
+    document.getElementById('cell1').innerHTML = '';
+    document.getElementById('cell2').innerHTML = '';
+    document.getElementById('cell3').innerHTML = '';
+    document.getElementById('cell4').innerHTML = '';
+    document.getElementById('cell5').innerHTML = '';
+    document.getElementById('cell6').innerHTML = '';
+    document.getElementById('cell7').innerHTML = '';
+    document.getElementById('cell8').innerHTML = '';
+    document.getElementById('winner-text').innerHTML = '';
```

Comment:
That approach is not good.
Please use a loop.

Reply:
Solved as for [commit](https://github.com/firmoholanda/tictactoe-js/pull/4/commits/359e6106a078dfab4924808bcf6a2c92ae2c6c4e)